src/slider/types.js

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import type {OverrideT} from '../helpers/overrides';
+import {} from './constants';
+
+export type OverridesT = {
+  Root?: OverrideT<*>,
+  Axis?: OverrideT<*>,
+  Tick?: OverrideT<*>,
+  TickBar?: OverrideT<*>,
+  Thumb?: OverrideT<*>,
+};
+
+export type PropsT = {
+  value: Array<*>,
+  range: Array<*>,
+  overrides?: OverridesT,
+  tabIndex: number,
+  error: boolean,
+  autoFocus: boolean,
+  disabled?: boolean,
+  $theme?: *,
+  onChange: (e: SyntheticEvent<HTMLInputElement>, params: ParamsT) => void,
+  onMouseEnter: (e: SyntheticEvent<HTMLInputElement>) => void,
+  onMouseLeave: (e: SyntheticEvent<HTMLInputElement>) => void,
+  onMouseDown: (e: SyntheticEvent<HTMLInputElement>) => void,
+  onMouseUp: (e: SyntheticEvent<HTMLInputElement>) => void,
+  onFocus: (e: SyntheticEvent<HTMLInputElement>) => void,
+  onBlur: (e: SyntheticEvent<HTMLInputElement>) => void,
```

Comment:
Can we leave off all these extra handlers besides `onChange`? From the discussion in #372 it sounds like @abmai is also on board with this change. Unless there is some special behavior that we provide beyond what the user would receive by using `overrides={{Root: {props: {onMouseEnter: ...}}}}`.

Reply:
fixed